accounts-db/src/accounts_hash.rs

```diff
@@ -994,54 +1010,152 @@ impl<'a> AccountsHasher<'a> {
                 }
             })
             .sum::<usize>();
+
+        (working_set, max_inclusive_num_pubkeys)
+    }
+
+    /// Add next item into hash dedup working set
+    fn add_next_item<'b>(
+        next: &mut Option<ItemLocation<'b>>,
+        working_set: &mut Vec<SlotGroupPointer>,
+        sorted_data_by_pubkey: &[&'b [CalculateHashIntermediate]],
+        pubkey_bin: usize,
+        binner: &PubkeyBinCalculator24,
+    ) {
+        // looping to add next item to working set
+        while let Some(ItemLocation {
+            key,
+            division_index: slot_group_index,
```

Comment:
seems like these are the same thing:
division_index: slot_group_index,

I'm sure my original terminology was confusing.

If they're the same, then one struct should be within the other struct:
```
ItemLocation<'a> {
pubkey,
pointer: WhateverPointer,
```

Thoughts?